wasm-text.md

```diff
@@ -109,46 +109,6 @@ In the future should investigate which direction the subsort should go.
     rule WasmIntToken2Int(VAL) => WasmIntTokenString2Int(replaceAll(WasmIntToken2String(VAL), "_", ""))
 ```
 
-### Folded Instructions
-
-Folded instructions are a syntactic sugar where expressions can be grouped using parentheses for higher readability.
-
-```k
-    syntax Instr ::= FoldedInstr
- // ----------------------------
-```
-
-One type of folded instruction are `PlainInstr`s wrapped in parentheses and optionally includes nested folded instructions to indicate its operands.
-
-```k
-    syntax FoldedInstr ::= "(" PlainInstr Instrs ")"
-                         | "(" PlainInstr        ")" [prefer]
- // ---------------------------------------------------------
-    rule <instrs> ( PI:PlainInstr IS:Instrs ):FoldedInstr => sequenceInstrs(IS) ~> PI ... </instrs>
-    rule <instrs> ( PI:PlainInstr           ):FoldedInstr =>                       PI ... </instrs>
-```
-
```

Comment:
@hjorthjort I don't understand. Aren't folded instructions part of the textual definition of Wasm?

Reply:
This got moved, the syntax productions are further down.
But the semantic rules get removed because we unfold them during pre-processing.